src/main/java/com/leverx/learningmanagementsystem/btp/servicemanager/config/ServiceManagerConfiguration.java

```diff
@@ -3,10 +3,12 @@
 import lombok.Data;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 
 @Configuration
 @ConfigurationProperties("service-manager")
 @Data
+@Profile("cloud")
```

Comment:
rename `smUrl` to `url` and `url` to `tokenUrl`, sometimes variables for different services are called in not very obvious way, but in our code it's better to resolve this problem